src/arvevaluator.c

```diff
@@ -508,6 +508,13 @@ arv_get_next_token (char **expression, gboolean previous_token_was_operand, gboo
 	return token;
 }
 
+double round_with_precision(double x, gint64 precision)
```

Comment:
Make it static in order to not leak function outside of arvevaluator.c
Please use the coding style